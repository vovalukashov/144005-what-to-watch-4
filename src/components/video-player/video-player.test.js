import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const playerProps = {
  src: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  poster: `https://images-na.ssl-images-amazon.com/images/I/81nwnHTcV2L._AC_SY679_.jpg`,
  muted: true,
  timeout: 1000,
  style: {objectFit: `cover`, width: `280px`, height: `175px`}
};

describe(`VideoPlayer snapshot`, () => {
  it(`VideoPlayer renderer`, () => {
    const tree = renderer
      .create(<VideoPlayer isPlaying={true} {...playerProps}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
