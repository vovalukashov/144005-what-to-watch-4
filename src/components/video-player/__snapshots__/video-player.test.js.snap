// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer snapshot VideoPlayer renderer 1`] = `
<video
  muted={true}
  poster="https://images-na.ssl-images-amazon.com/images/I/81nwnHTcV2L._AC_SY679_.jpg"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  style={
    Object {
      "height": "175px",
      "objectFit": "cover",
      "width": "280px",
    }
  }
  timeout={1000}
/>
`;
